src/main/java/com/fryrank/model/UserMetadata.java

```diff
@@ -7,7 +7,6 @@
 @Document("user-metadata")
 @Data
 public class UserMetadata {
-    @Id
```

Comment:
We don't want to change this because accountID should be put in the _id field in the Mongo collection. We should create a new object if we want to return user metadata that has a null accountId.